airgun/views/contentview_new.py

```diff
@@ -0,0 +1,62 @@
+from wait_for import wait_for
+from widgetastic.widget import Checkbox
+from widgetastic.widget import Text
+from widgetastic.widget import TextInput
+from widgetastic_patternfly4.ouia import Button as PF4Button
+from widgetastic_patternfly4.ouia import ExpandableTable
+
+from airgun.views.common import BaseLoggedInView
+from airgun.views.common import SearchableViewMixinPF4
+
+
+class NewContentViewTableView(BaseLoggedInView, SearchableViewMixinPF4):
+    title = Text('.//h1[contains(@data-ouia-component-id, "OUIA-Generated-Text")]')
+    create_content_view = PF4Button('create-content-view')
+    table = ExpandableTable(
+        component_id='content-views-table',
+        column_widgets={
```

Comment:
Are any of these actually widgets? If they're all Text, there should be no need to define column_widgets.

Reply:
This is from the old structure.  I'm not sure what how to construct the "correct" way one as the column widgets are for columns that have links, which these do.

Reply:
Ah! ack, the widgets are needed to make the cell content clickable, cool.